### ISSUE TITLE
Modal show active nav-links in nav-pills

### ISSUE THREAD

**Issue by danijelGombac** [twbs/bootstrap#22419]
v4.

```
.nav-pills .nav-link.active,
.show .nav-pills .nav-link {
  color: #fff;
  background-color: #0275d8;
}
```

This code add acitve state to all nav-links in nav-pill  when is in modal.show.

**Comment by Johann-S** [comment]
Bug reports must include a live demo of the problem. Per our contributing guidelines, please create a reduced test case via [JS Bin](https://jsbin.com/) or [CodePen](http://codepen.io/) and report back with your link, Bootstrap version, and specific browser and OS details.

**Comment by danijelGombac** [comment]
If look the code, you see the problem, there no need for live demo. When modal is open the class show is added to modal class="modal show". Now i don't have time to include live demo.

**Comment by danijelGombac** [comment]
How can i add v4-dev branch to JS Bin. From v4.0.0-alpha.6 this line is changed.

**Comment by danijelGombac** [comment]
There is also problem when .nav-pills is in div collapse and show is added.

**Comment by Johann-S** [comment]
You can copy/paste the content of `bootstrap.min.css` and `bootstrap.min.js` in your CodePen

**Comment by danijelGombac** [comment]
https://jsbin.com/doguyaruwe/edit?html,css,js,output

**Comment by mdo** [comment]
And that's why v3 was riddled with `>` for our selectors. :grin Fix coming in #22859.

<img width="642" alt="screen shot 2017-06-17 at 9 42 12 pm" src="https://user-images.githubusercontent.com/98681/27258091-da2f7a26-53a5-11e7-805c-ed804d8dd047.png">
